ui/raidboss/raidboss_config.ts

```diff
@@ -527,7 +527,7 @@ const setOptionsFromOutputValue = (
     options.SoundAlertsEnabled = false;
     options.SpokenAlertsEnabled = false;
   } else {
-    console.error(`unknown output type: ${value.toString()}`);
+    console.error(`unknown output type: ${JSON.stringify(value)}`);
```

Comment:
The linked PR doesn't mention the exact lint errors. What's the error that's being fixed with this change?

Reply:
```log
\cactbot-workspace\cactbot\ui\raidboss\raidboss_config.ts
   530:43  error  'value' may evaluate to '[object Object]' when stringified     @typescript-eslint/no-base-to-string
   765:26  error  'trigFunc' may evaluate to '[object Object]' when stringified  @typescript-eslint/no-base-to-string
  1349:27  error  'result' may evaluate to '[object Object]' when stringified    @typescript-eslint/no-base-to-string
```

I am not sure, but ESLint keep complaining that you cannot just `.toString()` it since the value is possibly an object?

Reply:
Alright, I see the issue here. I think this would benefit from a TODO or a FIXME comment, but otherwise the change to `JSON.stringify` makes sense.

Either this setter func ***can*** receive a nested object type as defined by the typedef for `SavedConfigEntry`, at which point it should handle that nested type properly, or it can't receive a nested type and the typedef for `setOptionsFromOutputValue` and the underlying type `ConfigEntry` should have its `setterFunc` fixed at some point in the future.

Reply:
Okay, I will add a FIXME comment here and then back to fix it in the future PR.